internal/pkg/agent/application/coordinator/testdata/simple_config/expected/local-config.yaml

```diff
@@ -0,0 +1,71 @@
+fleet:
+  enabled: false
+  access_api_key: ""
+  protocol: http
+  host: localhost:5601
+  timeout: 10m0s
+  agent:
+    id: ""
+agent:
+  id: ""
+  download:
+    operatingsystem: ""
+    architecture: ""
+    sourceuri: https://artifacts.elastic.co/downloads/
+    targetdirectory: /Users/blake/Library/Caches/JetBrains/GoLand2023.1/tmp/GoLand/data/elastic-agent-unknow/downloads
```

Comment:
these paths are not portable, they should be replaced with placeholder in the yaml and the hook actual output should be sanitized before comparison.
Have a look at [this snippet](https://github.com/elastic/elastic-agent/blob/1ec8d1c21bcc8bff6e0dfcb5be26289b230bc243/internal/pkg/agent/application/coordinator/diagnostics_test.go#L354) done for the `*-config.yaml` files

Reply:
I used the suggestion below to adjust the configuration to have static paths. That allows the test to run without having to call sanitize for the resulting hook.